spec/rails_helper.rb

```diff
@@ -18,7 +18,7 @@
   Capybara.javascript_driver = :selenium_chrome_headless
 end
 
-Capybara.default_max_wait_time = 5
+Capybara.default_max_wait_time = 3
```

Comment:
Initially when I started doing all this work I played around with increasing this time to 10 seconds just to see if it would help the tests pass. It didn't have any affect and the test would always fail on it comparing the expected text with the previous page (incorrectly) after a continue. So the wait didn't help it to grab the new correct page. 
This makes me think this is not working in the way we expect and we could actually decrease this time. But I'm still very confused on how this wait time should work everything I can find online about it seems to say something different than how I see it working here.